src/run-tests/Makefile

```diff
@@ -1,15 +1,15 @@
 .PHONY: test setup venv integ clean
 
 test: integ
+ 
+venv:
+	test -d venv || (python3 -m venv .venv)
 
 setup: venv
 	bash ./install-requirements-integ.sh
 
-venv:
-	test -d venv || (python3 -m venv .venv)
-
-integ: setup
-	bash ./run-tests-integ.sh $(service)
+integ: venv
+	bash ./run-tests-integ.sh $(SERVICE)
```

Comment:
Note that switch dependency from `setup` to `venv`

This is to avoid `pip install` every time we run tests.

As a result, we need to explicitly run `make setup` once before running `make test` later.